odl/discr/discr_mappings.py

```diff
@@ -444,13 +425,14 @@ def nearest(arg, out=None):
 
     def __repr__(self):
         """Return ``repr(self)``."""
-        inner_str = '\n  {!r},\n  {!r},\n  {!r}'.format(self.range,
-                                                        self.grid,
-                                                        self.domain)
-        if self.order == 'F':
-            inner_str += ",\n  order='F'"
-        if self._variant == 'right':
-            inner_str += ",\n  variant='right'"
+        inner_str = '\n  {!r},\n  {!r},\n  {!r}'.format(
+            self.range, self.grid, self.domain)
+        sep = ',\n '
```

Comment:
We could look into [textwrap.TextWrapper](https://docs.python.org/2/library/textwrap.html#textwrap.TextWrapper) for that. It seems to have enough flexibility to do what we want. It's slightly out of scope for this PR, I'll make a new issue.